src/components/banner/banner.scss

```
@@ -1,12 +1,18 @@
-@import '@limetech/mdc-elevation/mixins';
-@import '@limetech/mdc-theme/mixins';
 @import '../../style/internal/mdc-variables';
+@import '../../style/mixins';
+
+/**
+ * @prop --banner-icon-fill-color: Color of the banner icon
+ * @prop --banner-icon-background-color: Background color of the banner icon
```

```suggestion
 * @prop --banner-icon-fill-color: Color of the banner icon.
 * @prop --banner-icon-background-color: Background color of the banner icon.
 * @prop --icon-background-color: DEPRECATED. Use `--banner-icon-background-color` instead.
```

Fixup for `feat(banner): make icon fill color customizable & reveal color variables in the docs`.

While `--icon-background-color` was never properly documented, it was visible in the example, which means people might have used it. Better to make it clear that it has been deprecated and what to use instead.

---

I have no specific opinion ;) Looks good.